xyz.stuerz.BilligSweeper.yml

```
@@ -0,0 +1,27 @@
+app-id: xyz.stuerz.BilligSweeper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: billig-sweeper
+rename-desktop-file: billig-sweeper.desktop
+rename-icon: billig-sweeper
+
+modules:
+  - name: billig-sweeper
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/riscygeek/billig-sweeper/archive/refs/tags/v1.0.tar.gz
+        sha256: 27d79a341c5b15041bd4d0ff135da19c25460f014af1551a3d3421d000eb19e1
+    builddir: true
+  - name: billig-sweeper-metainfo
+    buildsystem: simple
+    sources:
+      - type: file
+        path: xyz.stuerz.BilligSweeper.metainfo.xml
+    build-commands:
+      - install -vDm644 -t /app/share/metainfo/ xyz.stuerz.BilligSweeper.metainfo.xml
+
+finish-args:
+    - --socket=x11
+    - --device=dri
```

```suggestion
    - --device=dri
    - --share=ipc
```

(required for X11)

also I'll assume this doesn't support wayland.

---

Well for some reason it worked without --share=ipc, but sure I guess.
I use SDL2 and I forgot to tell it to use Wayland by default, but I'm gonna fix it in the next release, if it'll work.